rero_ils/modules/documents/views.py

```diff
@@ -78,7 +78,8 @@ def doc_item_view_method(pid, record, template=None, **kwargs):
         pid=pid,
         record=record,
         holdings=holdings,
-        viewcode=viewcode
+        viewcode=viewcode,
+        recordType='documents'
```

Comment:
why do you need this?

Reply:
`recordType` is used into the template file to create the 'next_url' (url used to redirect after document suppression). You can see that into `rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html:151`.
I didn't find any simple solution to determine the `recordType` depending of `record`.